Examples/Io/Root/include/ActsExamples/Io/Root/RootTrackParameterWriter.hpp

```diff
@@ -98,26 +95,61 @@ class RootTrackParameterWriter final : public TrackParameterWriter {
   std::mutex m_writeMutex;  ///< Mutex used to protect multi-threaded writes
   TFile* m_outputFile{nullptr};  ///< The output file
   TTree* m_outputTree{nullptr};  ///< The output tree
-  int m_eventNr{0};              ///< the event number of
+
+  int m_eventNr{0};  ///< the event number of
+
+  int m_volumeId{0};   ///< volume ID
+  int m_layerId{0};    ///< layer ID
+  int m_surfaceId{0};  ///< surface ID
```

Comment:
Should we get rid of the redundant comments and may be write before each block something like "Truth parameter" and so on?